README.md

```diff
@@ -533,13 +537,23 @@ brew install octave
 The `ROBOTOLOGY_USES_OCTAVE` option has never been tested on Windows.
 
 ### Configuration
-Add the `$ROBOTOLOGY_SUPERBUILD_ROOT/build/install/octave` directory to your [Octave path](https://www.gnu.org/software/octave/doc/interpreter/Manipulating-the-Load-Path.html).
+Add the `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/octave` directory to your [Octave path](https://www.gnu.org/software/octave/doc/interpreter/Manipulating-the-Load-Path.html).
 
 FAQs
 ====
 
 See also YCM documentation for [YCM's FAQs](http://robotology.github.io/ycm/gh-pages/master/manual/ycm-faq.7.html).
 
+## Python
+
+Support for this dependency is enabled by the `ROBOTOLOGY_USES_PYTHON` CMake option.
+
+### Configuration
+Add the `$ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX/lib/python2.7/dist-packages` to your `PYTHONPATH`. This is done automatically if you source the provided `setup.sh` script.
+
+### Check the installation
+The folder above should contain `_*.so` files which correspond to the generated python bindings. Open a python interpreter and try to import modules.
```

Comment:
Please specify the folder to which you are referring, for example "The folder mentioned in the configuration/previous section". 
Furthermore `_*.so` files are only present in *nix system, so I would probably just mention the `.py` files.